### PR TITLE
Update IBU Doc

### PR DESCRIPTION
**Pull request by browsell**
Update the IBU Doc.

### CODE REVIEW

File: docs/image-based-upgrade.md [hunk 1]
```diff
@@ -84,6 +94,7 @@ The target SNO has the following prerequisites:
   - Same performance tuning (i.e., reserved CPUs).
 - LCA operator must be deployed, version must be compatible with the seed.
 - The OADP operator is installed along with a DataProtectionApplication CR. OADP has connectivity to a S3 backend
+- The target cluster must have a dedicated partion configured for `/var/lib/containers`
```
Comment (leo8a) [inline docs/image-based-upgrade.md:97]:
```suggestion
- The target cluster must have a dedicated partition configured for `/var/lib/containers`
```

File: docs/image-based-upgrade.md [hunk 2]
```diff
@@ -55,25 +55,35 @@ TODO Insert the state transition diagram
 
 ## Handling Site Specific Artifacts
 
-Two mechanisms are provided for the admin to apply site specific artifacts to the SNO following the pivot to the new version.
+In order to optimize the downtime of IBU, the intent is to apply as many artifacts as possible to the seed image.
+The applications are considered site specific and are not part of this seed image.
+In addition, there are platform artifacts that are site and OCP version specific so cannot simply backed up and restored.
```
Comment (sakhoury) [inline docs/image-based-upgrade.md:60]: nit
```suggestion
In addition, there are platform artifacts that are site and OCP version specific so they cannot simply be backed up and restored.
```

File: docs/image-based-upgrade.md [hunk 3]
```diff
@@ -55,25 +55,35 @@ TODO Insert the state transition diagram
...
 backup and restore CR. Refer to [backuprestore-with-oadp](backuprestore-with-oadp.md).
 
+### Extra Manifests
+
+The Life Cycle Agent provides a mechanism to apply a set of extra manifests after booting the new OCP version.
+This is generally intended for platform configuration that is site specific and could change over OCP releases. It would also be used for new manifests that are only relevant in the new release.
+
+There are two implementations of extra manifests:
+
+- If the target cluster is integrated with ZTP Gitops, the site specific manifests can be automatically extracted by the operator during the upgrade stage. Manifests with a label of `lca.openshift.io/target-ocp-version: “4.y.x” will be extracted and applied after rebooting to the new version.
```
Comment (sakhoury) [inline docs/image-based-upgrade.md:82]:
```suggestion
- If the target cluster is integrated with ZTP GitOps, the site specific manifests can be automatically extracted by the operator during the upgrade stage. Manifests that have the `ran.openshift.io/ztp-deploy-wave` annotation and are labeled with `lca.openshift.io/target-ocp-version: “4.y.x” will be extracted and applied after rebooting to the new version.
```

File: docs/image-based-upgrade.md [hunk 4]
```diff
@@ -55,25 +55,35 @@ TODO Insert the state transition diagram
...
+
+- If the target cluster is not integrated with ZTP Gitops the extra manifests can be provided via configmap(s) applied to the cluster. These configmap(s) specified by the
```
Comment (sakhoury) [inline docs/image-based-upgrade.md:84]:
```suggestion
- If the target cluster is not integrated with ZTP GitOps the extra manifests can be provided via configmap(s) applied to the cluster. These configmap(s) specified by the
```

File: docs/image-based-upgrade.md [hunk 5]
```diff
@@ -55,25 +55,35 @@ TODO Insert the state transition diagram
...
 backup and restore CR. Refer to [backuprestore-with-oadp](backuprestore-with-oadp.md).
 
+### Extra Manifests
+
+The Life Cycle Agent provides a mechanism to apply a set of extra manifests after booting the new OCP version.
+This is generally intended for platform configuration that is site specific and could change over OCP releases. It would also be used for new manifests that are only relevant in the new release.
+
+There are two implementations of extra manifests:
+
+- If the target cluster is integrated with ZTP GitOps, the site specific manifests can be automatically extracted by the operator during the upgrade stage. Manifests that have the `ran.openshift.io/ztp-deploy-wave` annotation and are labeled with `lca.openshift.io/target-ocp-version: “4.y.x” will be extracted and applied after rebooting to the new version.
```
Comment (donpenney) [inline docs/image-based-upgrade.md:82]:
```suggestion
- If the target cluster is integrated with ZTP GitOps, the site specific manifests can be automatically extracted by the operator during the upgrade stage.
  Manifests that have the `ran.openshift.io/ztp-deploy-wave` annotation and are labeled with `lca.openshift.io/target-ocp-version: “4.y.x” will be extracted and applied after rebooting to the new version.
```

Comment (sakhoury) [inline docs/image-based-upgrade.md:82]:
```suggestion
- If the target cluster is integrated with ZTP GitOps, the site specific manifests can be automatically extracted by the operator during the upgrade stage. 
 Manifests that have the `ran.openshift.io/ztp-deploy-wave` annotation and are labeled with `lca.openshift.io/target-ocp-version: “4.y.x”` will be extracted and applied after rebooting to the new version.
```

**Review by sakhoury — COMMENTED** [review]
/lgtm

File: docs/image-based-upgrade.md (hunk 5 above)
Comment (sakhoury) [reply docs/image-based-upgrade.md:82]: Add missing backtick to:
> `lca.openshift.io/target-ocp-version: “4.y.x”`

**Pull request merged**